app/public/js/location_search_table.js

```diff
@@ -0,0 +1,40 @@
+function bindeUpdateRadius() {
+	preventSubmitForm();
+	$("#updateRadius").click(onclickUpdateButtonEvent);
+	function onclickUpdateButtonEvent(event){
+		var updateRadiusApi = "/api/set-search-radius"
+		var distNumber = parseFloat($("#distance").val());
+		if (!distNumber || distNumber > 200  || distNumber<=0 ) {
```

Comment:
Like in the server's API, remove the distNumber > 200  || part of this condition.